src/dials/util/export_shelx.py

```diff
@@ -167,3 +169,161 @@ def _write_ins(experiment_list, best_unit_cell, ins_file):
                 )
             )
         LATT_SYMM(f, sg)
+        if composition:
+            logger.info(
+                f"Using composition {composition} to write SFAC & UNIT instructions"
+            )
+            sorted_composition = _parse_compound(composition)
+            f.write("SFAC %s\n" % " ".join(sorted_composition))
+            f.write(
+                "UNIT %s\n"
+                % " ".join(
+                    [
+                        str(sorted_composition[e] * sg.order_z())
+                        for e in sorted_composition
+                    ]
+                )
+            )
+        else:
+            f.write("SFAC C\n")
+            f.write("UNIT 0\n")
+        f.write("HKLF 4\n")  # Intensities
+        f.write("END\n")
+
+
+def _parse_compound(composition):
+    """
+    Modified from parse_compund() in xia2/src/xia2/cli/to_shelx.py
+    """
```

Comment:
This I am sure is already available in `cctbx` somewhere else?

Reply:
yes - `cctbx/eltbx/chemical_elements.h` which I assume is exported somewhere

Reply:
Ahh yes - thanks!
```
from cctbx.eltbx.chemical_elements import proper_caps_list
proper_caps_list()[:94]
```
Can replace the list of elements with scattering factors encoded in SHELXL included here.